requirements/ci.txt

```diff
@@ -1,5 +1,5 @@
 -r wheel.txt
-pip==9.0.1
+pip==9.0.2
```

Comment:
@asvetlov I didn't realize we've got pip in requirements. I'd manage its versions separately.

Reply:
Agree